ui-tests/src/test/resources/features/integrations/slack.feature

```diff
@@ -160,23 +161,26 @@ Feature: Slack Connector
     And checks that query "select * from contact where company = 'message4' AND first_name = 'syndesis-bot'" has "1" output
 
     # test delay
-    Then send message "messageDelayed" on channel "random"
+    Then send message "messageDelayed" on channel "slack_connector_test"
     And checks that query "select * from contact where company = 'messageDelayed'" has no output
-    And sleep for "60000" ms
+    And sleep for "30000" ms
     Then checks that query "select * from contact where company = 'messageDelayed' AND first_name = 'syndesis-bot'" has "1" output
 
     # test Maximum Messages to Retrive after delay
-    Then send message "message5" on channel "random"
-    And send message "message6" on channel "random"
-    And send message "message7" on channel "random"
-    And send message "message8" on channel "random"
-    And sleep for "60000" ms
+    Then send message "message5" on channel "slack_connector_test"
+    And send message "message6" on channel "slack_connector_test"
+    And send message "message7" on channel "slack_connector_test"
+    And send message "message8" on channel "slack_connector_test"
+    And sleep for "30000" ms
     #After first delay it should consume only two messages (Max)
     Then checks that query "select * from contact where company = 'message5' AND first_name = 'syndesis-bot'" has "1" output
     And checks that query "select * from contact where company = 'message6' AND first_name = 'syndesis-bot'" has "1" output
     And checks that query "select * from contact where company = 'message7'" has no output
     And checks that query "select * from contact where company = 'message8'" has no output
-    And sleep for "60000" ms
+    And sleep for "15000" ms
+    And checks that query "select * from contact where company = 'message7'" has no output
+    And checks that query "select * from contact where company = 'message8'" has no output
+    And sleep for "150000" ms
```

Comment:
does this need such a long wait?

Reply:
I have already reduce it on half :D, I added there the long wait for sure that everything has been done on external service (slack) before checking. I can try to test it with more reduction, let's say 5 seconds.

Reply:
Sure, was just asking, in case it's relevant, you can leave as it is - the main concern there was related to the current effort to reduce the total test execution time.

Reply:
The 150000 was a TYPO, it should be 15000. The decrease of time more will cause the test is not stable.